src/lib/utils.ts

```diff
@@ -968,7 +968,7 @@ export function forEachMonth(
   let current = start;
   while (current.isSameOrBefore(end, "month")) {
     cb(current);
-    current = current.add(1, "month");
+    current = current.startOf("month").add(1, "month");
```

Comment:
This can be moved out of loop and done once on the start? (line 968)

Reply:
Fixed